core/src/main/java/com/linecorp/armeria/common/HttpResponseWriter.java

```diff
@@ -199,4 +199,113 @@ default void respond(AggregatedHttpMessage res) {
 
         close();
     }
+
+    /**
+     * Convenience method for redirect.
+     *
+     * @param format {@linkplain Formatter the format string} of the response content
+     * @param args the arguments referenced by the format specifiers in the format string
+     */
+    default void redirect(String format, Object... args) {
+        requireNonNull(format, "format");
+        requireNonNull(args, "args");
+        redirect(HttpStatus.TEMPORARY_REDIRECT, format, args);
+    }
+
+    /**
+     * Convenience method for redirect.
+     *
+     * @param newLocation the new location
+     */
+    default void redirect(String newLocation) {
+        requireNonNull(newLocation, "newLocation");
+        redirect(HttpStatus.TEMPORARY_REDIRECT, newLocation);
```

Comment:
assume that default redirect use 302 seems not a very nature assumption, how about remove this method and method at line 212 and remain the explicit (whith HttpStatus parameter) only

Reply:
Ah, looks line @trustin already commented the same thing.